app/models/campaign_gift_option.rb

```diff
@@ -13,13 +13,12 @@ class CampaignGiftOption < ApplicationRecord
 		:order, #int (optional) 
 		:hide_contributions #boolean (optional) 
 
-	belongs_to :campaign
+	belongs_to :campaign, required: true
```

Comment:
The shoulda-matchers work slightly different in Rails 5. Even a belongs_to is validated as present, the matcher won't work unless I set required. Don't know why.